.github/pr_title_checker_config.json

```diff
@@ -1,6 +1,6 @@
 {
     "LABEL": {
-        "name": "title needs formatting",
+        "name": "[pr-title] Should be like this",
```

Comment:
```suggestion
        "name": "PR title is missing [component]",
```

Maybe something like this?

Reply:
I believe this is a label for the check (see table in the bottom of this PR). It should not be an error.

Reply:
Yea, I realize this. Just think `[pr-title] Should be like this` is a confusing label.

Reply:
```suggestion
        "name": "PR title format",
```
Might be too generic but I suspect we might add more checks later (e.g. length)